googlemock/src/gmock-spec-builders.cc

```diff
@@ -487,8 +487,24 @@ bool UntypedFunctionMockerBase::VerifyAndClearExpectationsLocked()
       // takes care of it.
       untyped_expectation->MaybeDescribeExtraMatcherTo(&ss);
       untyped_expectation->DescribeCallCountTo(&ss);
-      Expect(false, untyped_expectation->file(),
-             untyped_expectation->line(), ss.str());
+      if (! registry_destructed_) {
```

Comment:
???

(You probably missed the previous replay about that: [ref](https://github.com/google/googletest/pull/653#discussion_r46755716).
I'll repeat it below, since your suggestion is not clear to me.).

>if (registry_destructed_) case can go first

That if is actually in a for-loop. One probably should not cut short the iterations by an immediate return (return is also bool, not void).

My thinking was:
Use a conditional, to choose between Expect or std::cout.
Note also: Both branches (if and else) rely on common code (::std::stringstream ss; ...)

(Am I missing something?)